documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -96,6 +96,41 @@ spec:
 .Additional resources
 For more information, refer to the xref:type-BrokerCapacity-reference[].
 
+== Topic creation and configuration
+
+Cruise Control requires the following three topics to be created in order to function properly:
+
+* `strimzi.cruisecontrol.partitionmetricsamples`
+* `strimzi.cruisecontrol.modeltrainingsamples`
+* `strimzi.cruisecontrol.metrics`
+
+Cruise Control will attempt to create these topics automatically.
+However, if automatic topic creation is disabled in Kafka with a configuration of `auto.create.topics.enable: false` in `spec.kafka.config`, Cruise Control will be unable to create these topics.
+In this situation, the Cruise Control topics will need to be created manually using the `example/topic/cruise-control-topics.yaml` file.
```

Comment:
Stupid question - but CO already uses the AdminClient and has access to the brokers. So why not manage the topic form the operator using AdminClient our self?

Reply:
I agree with Jakub. As we discussed offline, it could be a metrics reporter bug not able to keep trying to create the needed topics while the Kafka cluster is starting up. It needs a little bit more investigation in the CC codebase. Anyway, if fixing the CC is a long way, doing the creation of the topics in the CO could be a better idea than document and asking users to do so.

Reply:
Paolo and I found that this can be fixed upstream by making the CC metric reporter AdminClient timeout configurable. We could also add logic to the CO that would take care of this for us